zubhub_frontend/zubhub/src/components/Navbar/Navbar.jsx

```diff
@@ -77,6 +77,8 @@ function PageWrapper(props) {
   const formRef = useRef();
   const token = useSelector(state => state.auth.token);
 
+  console.log("TOken", token);
```

Comment:
Good job, you forgot to delete this 🙂

Reply:
oooooh, that's true. Thanks for noting that @brrkrmn , added that when trying to debug when the projects weren't showing. Let me remove it.